src/dsp_tools/commands/validate_data/prepare_data/get_rdf_like_data.py

```diff
@@ -91,6 +96,51 @@ def _get_generic_value(value: ParsedValue, user_value: str | None) -> RdfLikeVal
     )
 
 
+def _get_date_value(value: ParsedValue) -> RdfLikeValue:
+    typed_val: str | None = value.value if isinstance(value.value, str) else None
+    date_metadata = _get_value_metadata(value)
+    if typed_val:
+        date_metadata.extend(_get_xsd_like_dates(typed_val))
+    return RdfLikeValue(
+        user_facing_prop=value.prop_name,
+        user_facing_value=typed_val,
+        knora_type=value.value_type,
+        value_metadata=date_metadata,
+    )
+
+
+def _get_xsd_like_dates(date_string: str) -> list[PropertyObject]:
+    parsed_date = parse_date_string(date_string)
+    dates = []
+    if not (ce_start := _make_xsd_compatible_date(parsed_date.start, TriplePropertyType.KNORA_DATE_START)):
+        return []
+    dates.append(ce_start)
+    if parsed_date.end:
+        if not (ce_end := _make_xsd_compatible_date(parsed_date.end, TriplePropertyType.KNORA_DATE_END)):
+            return []
```

Comment:
Just to double-check: If the end date should be there, but cannot be parsed, you don't return the (successfully parsed) start date, but an empty list?